src/CommonITILTask.php

```diff
@@ -650,7 +650,7 @@ public function post_addItem()
             }
         }
 
-        $this->input["_job"]->updateDateMod($this->input[$this->input["_job"]->getForeignKeyField()]);
+        $this->input["_job"]->updateDateMod($this->input[$this->input["_job"]->getForeignKeyField()], $this->input['_do_not_compute_takeintoaccount'] ?? false);
```

Comment:
This change should probably de done to all the calls to `updateDateMod`. This same ability to not alter SLA/OLA stats will probably be usefull for other actions, like adding an actor, updating a task, adding an automatic followup, ...